crates/testsuite/features/cmd.feature

```diff
@@ -13,7 +13,7 @@ Feature: Rooch CLI integration tests
       # TODO split Scenario for every example
       # counter example
       Then cmd: "move publish -p ../../examples/counter --sender-account {default} --named-addresses rooch_examples={default}"
-      Then cmd: "move run --function {default}::counter::init --sender-account {default}"
+      Then cmd: "move run --function {default}::counter::init_entry --sender-account {default}"
```

Comment:
there is an example dedicated to verifying module init, use this example `module_init_example `?